conformance/lib/build.gradle.kts

```diff
@@ -69,4 +89,12 @@ tasks {
         kotlinPackage.set("pbandk.conformance.pb")
         logLevel.set("debug")
     }
+
+    // DCE is now enable by default in Kotlin 1.3.70/71
+    // and it doesn't work well with commonJS modules
+    // Use of commonJs could be removed since default module is now UMD
+    // but would require some code change
+    val processDceJsKotlinJs by getting {
```

Comment:
I did want to keep commonJs package here like what it was before but now JS are bundle as UMD which is compatible. Any prefrence?

Reply:
I'm not a JS expert, but it seems like anything that works with CommonJS will work with UMD, so let's switch to UMD for pbandk.

Reply:
That's what I read too. I'll switch to UMD

Reply:
I did keep commonJS because UMD involved naming with JSModule and some other things... It's working now so maybe a change in the future.